examples/getting-started-typescript/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "getting-started-typescript",
+  "version": "1.0.0",
+  "description": "Bolt getting started app in TypeScript",
+  "main": "dist/app.js",
+  "scripts": {
```

Comment:
I know you picked these up from my example. Many developers may eventually want to use webpack etc but I think it's fine to provide this simple example as the very first step.